recipe/conda_build_config.yaml

```
@@ -6,6 +6,6 @@ MACOSX_SDK_VERSION:        # [osx and x86_64]
 
 # Build only for CUDA 11.2+
 cuda_compiler_version_min:
-  - None                       # [osx]
-  - 11.2                       # [linux64 or win]
-  - 11.2                       # [linux and (ppc64le or aarch64)]
+  - None
+  - 11.2                       # [(linux64 or win) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - 12.0                       # [(linux64 or win) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
```

This should fix it so that only one "min" value is seen for each build.
```suggestion
  - None  # [not (linux64 or ppc64le or aarch64)]
  - 11.2  # [linux64 or ppc64le or aarch64]
```

---

I had to revert back to the original 3 item list. `conda-smithy rerender` chokes on the 2 item list.